### PR TITLE
Migrate to latest version of Selenium + Firefox #3643

### LINKED ISSUE #3643: Migrate to latest version of Selenium + Firefox

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #3643 
Download Selenium [here](http://selenium-release.storage.googleapis.com/index.html?path=2.46/), only `selenium-java-2.46.0-srcs.jar` (found inside the Java zip) and `selenium-server-standalone-2.46.0.jar` are required. Place them on `src/test/resources/lib/selenium/`.
Update Firefox to the latest one (38.0.5) [here](https://www.mozilla.org/en-US/firefox/new/).
Eclipse will show a few warnings due to deprecations and what-nots but everything will still work.

### CODE REVIEW

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -255,7 +244,7 @@ Troubleshooting instructions are given [in this document](https://docs.google.co
     TestNG is a Java test automation framework.
 * **QUnit** [version 1.10.0]
     QUnit is a JavaScript unit test suite.
-* **NekoHtml** [version 1.9.16]
+* **NekoHtml** [version 1.9.21]
```
Comment (damithc) [inline devdocs/settingUp.md]: This got updated together with Selenium?

Comment (wkurniawan07) [inline devdocs/settingUp.md]: Yes, but I got the version wrong. It should be 1.9.22. Will fix.

**Comment by kai33** [comment]
is InstructorFeedbackResultsPage.java related to this pull request?

**Comment by wkurniawan07** [comment]
Yes, in particular where I add this line:
`jsExecutor.executeScript("arguments[0].scrollIntoView(true);", photoDiv);`
Before I added that, `instructorFeedbackResultsPageUiTest` fails all the time. It seems that somehow the driver can't locate the element if it's not in the viewport.

**Comment by MrFool** [comment]
I'll try this out some time within the next few days and see if it's all good on Mac OS

**Comment by kanghj** [comment]
Works fine for me

**Comment by wkurniawan07** [comment]
@yj-soh @bamboo3250 @JosephineKwa @taniach your inputs will be appreciated!

**Comment by MrFool** [comment]
Just tested this on Mac OS, looks good :+1:

**Comment by damithc** [comment]
I guess we can go ahead and merge. Once merged, I need to update the zip file that contains all the jar files.

**Comment by wkurniawan07** [comment]
Ok, can always revert if unexpected things happen (which shouldn't happen... I hope). Just keep the old Selenium jars and Firefox 21 for a while.

**Comment by damithc** [comment]
If you haven't merged already, in the same PR you can update the documentation to point to this zip file http://www.comp.nus.edu.sg/~seer/teammates-libs/libsV5.46.zip

**Comment by wkurniawan07** [comment]
Alright sure

**Comment by bamboo3250** [comment]
it seems work fine on Windows 7 as well

**Comment by taniach** [comment]
Works well on Mac!

**Pull request merged**